db/sltdbt.c

```diff
@@ -243,6 +243,82 @@ int init_opcode_handlers()
     return 0;
 }
 
+static inline int opcode_supported(int opcode)
+{
+    static int last_rej = 0;
+    static unsigned long long rejcnt = 0;
+    int now;
+
+    if (!gbl_rowlocks && !gbl_disable_tagged_api)
```

Comment:
Why do we need to check for rowlocks?

Reply:
I guess that because we don't want to support all the tagged operations when rowlocks is enabled.

Reply:
This part of the patch was copied from R6.